docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -0,0 +1,212 @@
+
+    kubectl describe pod liveness-exec
+
+The output indicates that no liveness probes have failed yet:
+
+    FirstSeen    LastSeen    Count   From            SubobjectPath           Type        Reason      Message
+    --------- --------    -----   ----            -------------           --------    ------      -------
+    24s       24s     1   {default-scheduler }                    Normal      Scheduled   Successfully assigned liveness-exec to worker0
+    23s       23s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulling     pulling image "gcr.io/google_containers/busybox"
+    23s       23s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulled      Successfully pulled image "gcr.io/google_containers/busybox"
+    23s       23s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Created     Created container with docker id 86849c15382e; Security:[seccomp=unconfined]
+    23s       23s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Started     Started container with docker id 86849c15382e
+
+After 30 seconds, view the Pod events again:
+
+    kubectl describe pod liveness-exec
+
+At the bottom of the output, there are messages indicating that the liveness
+probes have failed, and the containers have been killed and recreated.
+
+    FirstSeen LastSeen    Count   From            SubobjectPath           Type        Reason      Message
+  --------- --------    -----   ----            -------------           --------    ------      -------
+    37s       37s     1   {default-scheduler }                    Normal      Scheduled   Successfully assigned liveness-exec to worker0
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulling     pulling image "gcr.io/google_containers/busybox"
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulled      Successfully pulled image "gcr.io/google_containers/busybox"
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Created     Created container with docker id 86849c15382e; Security:[seccomp=unconfined]
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Started     Started container with docker id 86849c15382e
+    2s        2s      1   {kubelet worker0}   spec.containers{liveness}   Warning     Unhealthy   Liveness probe failed: cat: can't open '/tmp/healthy': No such file or directory
+
+Wait another minute, and verify that the Container has been restarted:
+
+    kubectl get pod liveness-exec
+
+The output shows that `RESTART` has been incremented:
+
+    NAME            READY     STATUS    RESTARTS   AGE
+    liveness-exec   1/1       Running   1          1m
+
+### Defining a liveness HTTP request
+
+Another kind of liveness probe uses and HTTP GET request. Here is the configuration
+file for a Pld that runs a container based on the `gcr.io/google_containers/liveness`
```

Comment:
is this a new image just for this doc?  Can we find a way to better name these?  The proliferation of images that exist for tests, docs, and demos makes it very hard to see whats up in that namespace.

If it pre-existed, I'll not make a scene :)